cloudant-client/src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -1010,6 +1010,59 @@ public boolean contains(String id) {
     }
 
     /**
+     * Removes an attachment from the specified document.
+     * <p>The object must have the correct {@code _id} and {@code _rev} values.</p>
+     * <P>Example usage:</P>
+     * <pre>
+     * {@code
+     * //get a Bar object from the database
+     * Bar bar = db.find(Bar.class, "exampleId");
+     * String attachmentName = "example.jpg";
+     * //now remove the remote Bar
+     * Response response = db.removeAttachment(bar, attachmentName);
+     * }
+     * </pre>
+     *
+     * @param object the document to remove as an object
+     * @param attachmentName the attachment name to remove
+     * @return {@link com.cloudant.client.api.model.Response}
+     * @throws NoDocumentException If the document is not found in the database.
+     * @see <a target="_blank" href="https://docs.cloudant.com/attachments.html#delete">Documents -
+     * delete</a>
+     */
+    public com.cloudant.client.api.model.Response removeAttachment(Object object, String attachmentName) {
+        Response couchDbResponse = db.removeAttachment(object, attachmentName);
+        com.cloudant.client.api.model.Response response = new com.cloudant.client.api.model
+                .Response(couchDbResponse);
+        return response;
+    }
+
+    /**
+     * Removes the attachment from a document the specified {@code _id} and {@code _rev} and {@code attachmentName}
+     * values.
+     * <P>Example usage:</P>
+     * <pre>
+     * {@code
+     * Response response = db.removeAttachment("exampleId", "1-12345exampleRev", "example.jpg");
+     * }
+     * </pre>
+     *
+     * @param id  the document _id field
+     * @param rev the document _rev field
+     * @param attachmentName the attachment name
+     * @return {@link com.cloudant.client.api.model.Response}
+     * @throws DocumentConflictException if the attachment cannot be saved because of a conflict
```

Comment:
I think `NoDocumentException` should be in the throws list for the documentation.